### PR TITLE
Add an API to get the underlying `LazyTensorOperation` of a `LazyTensorHandle`.

### PR DESCRIPTION
**Pull request by bgogul**

### CODE REVIEW

File: Tests/TensorFlowTests/LazyTensorHandleTests.swift [hunk 1]
```diff
@@ -52,6 +52,21 @@ final class LazyTensorHandleTests: XCTestCase {
         XCTAssertEqual(liveSymTensor.description, "%0.0*")
     }
 
+    func testLazyTensorOperationProperty() {
+        let zero = Tensor<Float>(0.0)
+        let zeroTFEHandle = zero.handle.handle._tfeTensorHandle
+        let concTensor = LazyTensorHandle(zeroTFEHandle)
+        XCTAssertTrue(concTensor.lazyTensorOperation == nil)
```
Comment (dan-zheng) [inline Tests/TensorFlowTests/LazyTensorHandleTests.swift]:
```suggestion
        XCTAssertNil(concTensor.lazyTensorOperation)
```

File: Tests/TensorFlowTests/LazyTensorHandleTests.swift [hunk 2]
```diff
@@ -52,6 +52,21 @@ final class LazyTensorHandleTests: XCTestCase {
...
+
+        let op = LazyTensorOperation(
+            _id: "0", name: "IdentityN", outputCount: 3)
+        let symTensor = LazyTensorHandle(_lazy: op, index: 0)
+        let lazyTensorOperation = symTensor.lazyTensorOperation
+        XCTAssertFalse(lazyTensorOperation == nil)
```
Comment (dan-zheng) [inline Tests/TensorFlowTests/LazyTensorHandleTests.swift]:
```suggestion
        XCTAssertNotNil(lazyTensorOperation)
```

File: Tests/TensorFlowTests/LazyTensorHandleTests.swift [hunk 3]
```diff
@@ -52,6 +52,21 @@ final class LazyTensorHandleTests: XCTestCase {
...
+        // Checks that returned value is the same as the one that we passed in.
+        XCTAssertTrue(lazyTensorOperation! === op)
```
Comment (rxwei) [inline Tests/TensorFlowTests/LazyTensorHandleTests.swift]: [`===(_:_:)`](https://developer.apple.com/documentation/swift/1538988) is defined as a function that takes optionals. Force unwrapping should be avoided.
```suggestion
        XCTAssertTrue(lazyTensorOperation === op)
```

Comment (bgogul) [reply Tests/TensorFlowTests/LazyTensorHandleTests.swift]: ah I see. Thanks!

File: Tests/TensorFlowTests/LazyTensorHandleTests.swift (hunk 1 above)
Comment (bgogul) [reply Tests/TensorFlowTests/LazyTensorHandleTests.swift]: Thanks for the suggestion! Fixed it!

**Pull request merged**